src/app/models/PlanModel.ts

```diff
@@ -0,0 +1,40 @@
+import * as mongodb from 'mongodb';
```

Comment:
```suggestion
import { ClassInterface } from './ClassModel';
const { defaultProfilePicture } = require('../../config/defaultVars.config');
```

Reply:
@jpatrici When I removed the mongodb import, I am now getting TS errors "cannot find namespace or name 'mongodb'". This affects lines 5 and 28. Is this an actual problem?

Reply:
I made a mistake in commenting, don't remove the mongodb import, was meant to just add the class interface and the defaultProfilePicture

Reply:
Okay! I will change it. I bet this fixes the date issue as well.